<ui version="4.0" >
 <class>LastFmConfigWidget</class>
 <widget class="QWidget" name="LastFmConfigWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>306</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>1</width>
    <height>1</height>
   </size>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item row="5" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="2" row="0" column="0" >
    <widget class="QGroupBox" name="groupBox3" >
     <property name="title" >
      <string>Last.fm Profile</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="1" column="0" >
       <widget class="QLabel" name="labelUsername" >
        <property name="text" >
         <string>&amp;Username:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>kcfg_ScrobblerUsername</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="labelPassword" >
        <property name="text" >
         <string>&amp;Password:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>kcfg_ScrobblerPassword</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="KLineEdit" name="kcfg_ScrobblerPassword" >
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="KLineEdit" name="kcfg_ScrobblerUsername" />
      </item>
      <item row="4" column="0" colspan="2" >
       <widget class="QLabel" name="kActiveLabel1" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>&lt;a href="http://www.last.fm:80/signup.php">Sign up to last.fm&lt;/a></string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QPushButton" name="testLogin" >
        <property name="text" >
         <string>&amp;Test Login</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="kActiveLabel2" >
     <property name="text" >
      <string>Why not join the &lt;a href="http://www.last.fm:80/group/Amarok+Users">Amarok last.fm group&lt;/a> and share your musical tastes with other Amarok users?</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QGroupBox" name="groupBox2" >
     <property name="title" >
      <string>Last.fm Services</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" >
      <item row="0" column="0" >
       <widget class="QCheckBox" name="kcfg_SubmitPlayedSongs" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>&amp;Submit tracks</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="kcfg_RetrieveSimilarArtists" >
        <property name="text" >
         <string>&amp;Retrieve similar artists</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_ScrobblerUsername</tabstop>
  <tabstop>kcfg_ScrobblerPassword</tabstop>
  <tabstop>testLogin</tabstop>
  <tabstop>kcfg_SubmitPlayedSongs</tabstop>
  <tabstop>kcfg_RetrieveSimilarArtists</tabstop>
 </tabstops>
 <includes>
  <include location="local" >klineedit.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
